config/test_glyphs-glyf_colr_1.py

```diff
@@ -2197,7 +2197,7 @@ def _get_glyph_definitions(position):
     return list(TestDefinitions().make_all_glyphs(position))
 
 
-def _build_font(names, position):
+def _build_font(names, position, no_cliplist):
```

Comment:
Personally I find “negative” bool variable names harder to read, consider renaming to have_cliplist or with_cliplist

Reply:
I flipped that around, and made one small change to the suffix order so that "_no_cliplist" always comes last.